includes/downloads/recalculations.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Recalculation functions for downloads.
+ * @package     EDD
+ * @subpackage  Downloads
+ * @copyright   Copyright (c) 2022, Easy Digital Downloads, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.1
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The hook is registered here because it's specifically for the cron job,
+ * but it calls the general download recalculation function.
+ */
+add_action( 'edd_recalculate_download_sales_earnings_deferred', 'edd_recalculate_download_sales_earnings' );
+
+/**
+ * Handles scheduling the recalculation for a download.
+ *
```

Comment:
Can we add a longer description on this as well that highlights a few things for developers:
1. If `WP_Cron` is not disabled, this will run automatically within 5 minutes of any recalculations being scheduled.
2. This is a possibly very expensive query to run, so only do it if you absolutely need to, otherwise rely on the deferred cron so that it doesn't affect load times after the change.